src/SWP/Bundle/WebRendererBundle/Theme/Configuration/TenantableConfigurationProvider.php

```diff
@@ -63,14 +63,19 @@ public function __construct(
      */
     public function getConfigurations()
     {
-        $configs = array_map(
-            [$this->loader, 'load'],
-            $this->fileLocator->locateFilesNamed($this->configurationFilename)
-        );
+        // Handle there being no theme.json file
+        try {
```

Comment:
yes listener, I meant that you need to improve the message in the `ExceptionListener` telling that there are not themes at all or that you need to install demo theme. It will be called later after the container is initializer in the theme context class where you throw an exception.